apps/tangle-dapp/containers/TransferTxContainer/TransferTxContainer.tsx

```diff
@@ -94,26 +108,44 @@ const TransferTxContainer: FC<TransferTxContainerProps> = ({
 
         <div className="p-9 flex flex-col gap-4">
           <Typography variant="body1" fw="normal">
-            Quickly transfer your {TANGLE_TOKEN_UNIT} tokens to a recipient on
+            Quickly transfer your {TANGLE_TOKEN_UNIT} tokens to a receiver on
             the Tangle Network. You can choose to send to either an EVM or a
             Substrate address.
           </Typography>
 
-          <BridgeInputGroup className="space-y-4 p-0 !bg-transparent">
-            <RecipientInput
-              className="bg-mono-20 dark:bg-mono-160"
-              onChange={(nextRecipientAddress) =>
-                setRecipientAddress(nextRecipientAddress)
-              }
-            />
+          <TxConfirmationRing
+            source={{
+              address: accAddress ?? '',
+              typedChainId: isEthereumAddress(accAddress ?? '')
+                ? PresetTypedChainId.TangleTestnetEVM
+                : PresetTypedChainId.TangleTestnetNative,
+            }}
+            dest={{
+              address: isValidReceiverAddress ? receiverAddress : '',
+              typedChainId: isEthereumAddress(receiverAddress)
+                ? PresetTypedChainId.TangleTestnetEVM
+                : PresetTypedChainId.TangleTestnetNative,
+            }}
+            title={`${amount ? amount : 0} ${TANGLE_TOKEN_UNIT}`}
```

Comment:
Ideally we'd want some kind of loading state instead of defaulting to use empty strings, otherwise we'd be ignoring the type system. Maybe using skeleton loader if `accAddress` is still not available?